.github/workflows/run-archiver.yml

```diff
@@ -24,14 +18,14 @@ on:
     - cron: "21 8 1 * *" # 8:21 AM UTC, first of every month
 
 jobs:
-  archive-run-small:
+  archive-run:
     defaults:
       run:
         shell: bash -l {0}
     strategy:
       matrix:
         # Note that we can't pass global env variables to the matrix, so we manually reproduce the list of datasets here.
-        dataset: ${{ fromJSON(format('[{0}]', inputs.small_runner || '"eia176","eia191","eia757a","eia860","eia860m","eia861","eia923","eia930","eiaaeo","eiawater","eia_bulk_elec","epacamd_eia","ferc1","ferc2","ferc6","ferc60","ferc714","mshamines","nrelatb","phmsagas"' )) }}
+        dataset: ${{ fromJSON(format('[{0}]', inputs.datasets || '"eia176","eia191","eia757a","eia860","eia860m","eia861","eia923","eia930","eiaaeo","eiawater","eia_bulk_elec","epacamd_eia","epacems",ferc1","ferc2","ferc6","ferc60","ferc714","mshamines","nrelatb","phmsagas"' )) }}
```

Comment:
Ugh I remember going through so many attempted workarounds with @e-belfer to try to make this less janky, but I think we're stuck with this.